plugins/avatar/client/src/components/Chat.jsx

```diff
@@ -176,9 +176,21 @@ export default function ChatBox({
           const arrayBuffer = await blob.arrayBuffer()
 
           self.startFromAudioFile(arrayBuffer)
-          setMessages((messages) => [...messages, agent + ": " + "hello"])
+          setMessages((messages) => [...messages, agent + ": " + value])
         })
 
+
+        // fetch('http://localhost:3030/api', 
```

Comment:
Is this code needed? Why is it commented?

Reply:
It is not needed for now.
Thank you